tests/ra_aid/test_provider_integration.py

```
@@ -12,7 +12,8 @@
     AnthropicStrategy,
     OpenAIStrategy,
     OpenAICompatibleStrategy,
-    OpenRouterStrategy
+    OpenRouterStrategy,
+    GeminiStrategy
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)